.devcontainer/Dockerfile

```diff
@@ -1,9 +1,13 @@
+FROM registry.hub.docker.com/pandoc/latex:3-ubuntu AS latex
+
 FROM mcr.microsoft.com/devcontainers/typescript-node:18
 
 RUN apt update \
     && export DEBIAN_FRONTEND=noninteractive \
-    && apt install -y libasound2 libgbm-dev libgtk-3-0 libatk-bridge2.0-0 libnss3 libxshmfence1 xvfb \
+    # For using tlaplus tools in the debug window
```

Comment:
`npm tests` returns the following error without installing libgtk:

```
@lemmy ➜ /workspaces/vscode-tlaplus (fix/add-java-and-latex-to-devcontainer) $ npm test

> vscode-tlaplus@1.6.0 pretest
> npm run compile


> vscode-tlaplus@1.6.0 compile
> tsc -p ./


> vscode-tlaplus@1.6.0 test
> node ./out/tests/runTest.js

Downloading VS Code 1.77.3 from https://update.code.visualstudio.com/1.77.3/linux-x64/stable: 1146869/124980454 
[...]
Downloading VS Code 1.77.3 from https://update.code.visualstudio.com/1.77.3/linux-x64/stable: complete
Downloaded VS Code 1.77.3 into /workspaces/vscode-tlaplus/.vscode-test/vscode-linux-x64-1.77.3
/workspaces/vscode-tlaplus/.vscode-test/vscode-linux-x64-1.77.3/VSCode-linux-x64/code: error while loading shared libraries: libgtk-3.so.0: cannot open shared object file: No such file or directory

Exit code:   127
Done

Failed to run tests: Failed
```

Reply:
Added the required libraries to run the tests from the terminal